meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/notepad-app.bb

```diff
@@ -0,0 +1,22 @@
+LICENSE = "CLOSED"
+LIC_FILES_CHKSUM = ""
+SRC_URI = "git://github.com/GENIVI/notepad-app.git;protocol=https \
+SRCREV = "75c9e0a86502b99d21c843ebd10e3a06a1093e15"
+
+S = "${WORKDIR}/git"
+
+DEPENDS = " qtdeclarative "
+RDEPENDS_${PN} = " persistence-client-library "
```

Comment:
Why this is RDEPENDS? I thought that it should be persistence-client-library if you are using this.